mcprep-build.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
Would be good to docstring the args at the top (even if this is mitigated by argparse once utilized)